packages/components/src/components/popover/popover-utils.ts

```diff
@@ -9,25 +9,34 @@ export type PopoverDirection = (typeof POPOVER_DIRECTIONS)[number];
 export const POPOVER_ARIA_ATTRIBUTES = ['aria-label'] as const;
 export type PopoverAriaAttribute = (typeof POPOVER_ARIA_ATTRIBUTES)[number];
 
-const safeZonePx = 16;
+export const safeZonePx = 16;
 
 export const updatePopoverStyles = (
   host: HTMLElement,
   spacer: HTMLDivElement,
   popover: HTMLDivElement,
   direction: PopoverDirection,
-  theme: Theme
+  theme: Theme,
+  isNative = false
```

Comment:
Maybe try to stick to the same order. Currently `theme` and `isNative` get flipped for the final parameters of `attachComponentCss`.